tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -127,3 +127,14 @@ def print(self,
     def _print(self, result: Mapping[str, Any]) -> Any:
         assert result["data"] is not None
         return result["data"]
+
+    @command
+    def set_viewport(self, context: str, viewport: Optional[Mapping[str, Any]] = None) -> Mapping[str, Any]:
+        params: MutableMapping[str, Any] = {"context": context}
+
+        if viewport is not None:
+            params["viewport"] = viewport
+        else:
+            params["viewport"] = None
```

Comment:
Sorry to mention it now but we should make that similar to the `SerializationOptions` arguments and class that is used in the `script` BiDi module.

Reply:
What does the use of the class like that enables?

Reply:
By using that class we could have tests that set only individual fields, but not accepting any kind of dict as in the current implementation. Means we exclude any extensible. Being in sync here with the script module would be good.